assets/tests/hooks/useSearchResults.test.ts

```diff
@@ -302,6 +302,19 @@ describe("useLimitedSearchResults", () => {
     )
     expect(result.current).toEqual(null)
   })
+
+  test("when no query is for empty string, returns null", () => {
+    const mockSocket = makeMockSocket()
+
+    const { result } = renderHook(() =>
+      useLimitedSearchResults(mockSocket, {
+        text: "",
+        property: "run",
+        limit: 5,
+      })
+    )
+    expect(result.current).toEqual(null)
+  })
```

Comment:
nit: I would prefer a newline between tests, though this is another thing that our formatter should just do >.>
```suggestion
  })

```